erica_app/erica/elster_xml/elster_xml_generator.py

```diff
@@ -303,7 +303,7 @@ def _add_vast_list_xml_nutzdaten(xml_top, version='7'):
     list_xml.set('version', version)
 
 
-def _add_vast_beleg_ids_request_nutzdaten(xml_top, user_data, year='2020'):
+def _add_vast_beleg_ids_request_nutzdaten(xml_top, user_data, year='2021'):
```

Comment:
Oh wow, even adapted the beleg request, nice. 👍 I wouldn't have thought of that.